fvm/evm/emulator/state/stateDB.go

```diff
@@ -481,6 +517,11 @@ func (db *StateDB) Commit(finalize bool) (hash.Hash, error) {
 	return updateCommit, nil
 }
 
+// This is a no-op for our custom implementation of the StateDB interface,
+// since Commit() already handles finalization and deletion of empty
+// objects.
+func (db *StateDB) Finalise(deleteEmptyObjects bool) {}
```

Comment:
This is a no-op in our case, because our custom implementation of `StateDB`, already handles the finalization and deletion of empty objects as part of [Commit](https://github.com/onflow/flow-go/blob/master/fvm/evm/emulator/state/stateDB.go#L350-L351) .